### PR TITLE
Do not export stale data, by using a watchdog timer

### PR DESCRIPTION
**Pull request by clonejo**

### CODE REVIEW

File: sml_exporter/__init__.py [hunk 1]
```diff
@@ -168,6 +172,27 @@ def event(self, message_body: SmlSequence) -> None:
             logger.debug(
                 "Vendor or device identifiers not initialized, event was ignored."
             )
+        self.reset_watchdog()
+
+    def clear_gauges(self):
+        logger.warning("Timeout on receiving serial data, clearing gauges.")
```
Comment (mweinelt) [inline sml_exporter/__init__.py]: We should probably talk about metrics here and not specify the concrete type.

Comment (clonejo) [reply sml_exporter/__init__.py]: Indeed, fixed.

File: sml_exporter/__init__.py [hunk 2]
```diff
@@ -168,6 +172,27 @@ def event(self, message_body: SmlSequence) -> None:
             logger.debug(
                 "Vendor or device identifiers not initialized, event was ignored."
             )
+        self.reset_watchdog()
+
+    def clear_gauges(self):
```
Comment (mweinelt) [inline sml_exporter/__init__.py]:
```suggestion
    def clear_metrics(self):
```

Also here.

Comment (clonejo) [reply sml_exporter/__init__.py]: D'oh!

**Pull request merged**